data/data/ovirt/masters/variables.tf

```diff
@@ -66,4 +66,15 @@ variable "ovirt_master_affinity_groups" {
 variable "ovirt_affinity_group_count" {
   type        = string
   description = "create a dependency between affinity_group module to masters module"
-}
\ No newline at end of file
+}
+
+variable "ovirt_master_auto_pinning_policy" {
+  type        = string
+  default     = "disabled"
```

Comment:
Why not set the `MasterAutoPinningPolicy` field in the tfvars config to `omitempty` so that the default value in  data/data/ovirt/variables-ovirt.tf is used? That would also obviate the need for the conditional in data/data/ovirt/masters/main.tf.